WestBayResidential/custom_mods
# Hotfix-columncheckbox

# What Changed
This pull request enables the use of the "check all" checkbox for the bulk staff-enrollment function. It integrates the JQery plugin DataTables and uses those functions to select all of the entries (employees) in a single column (a designated course). The specific custom functions that are triggered by a change in the topmost checkbox are found in the file `/custom_mods/staff/jquery/source/dt_check_enrol.js`.

Also included in this changeset is a confirmation page that is returned when the "Submit" button is clicked that lists all of the enrollees by name and course.

# How To Test
Log in to WBLMS as an 'administrator', and select from the right-side menu, under the "Management" section Manage staff enrollment > Select a category (from drop down list) > Click submit button. 

On the next page, click the top box for any of the listed courses. All of the checkboxes in that column should be immediately checked. Note the list of employee names that are visible on the page. Click "Enroll staff" button.

On the next page should be a list of the employee names that were previously viewed, with messages indicating that each of those employees has been enrolled in the selected course.
